### ISSUE TITLE
Eval/classifier is too small, and synthesised speech unclear.

### ISSUE THREAD

**Issue by mudong0419** [Tomiinek/Multilingual_Text_to_Speech#56]
Thanks for your excellent work. I'm trying to synthesis Chinese-English code-switched speech. Datasets are as followed:
ST-CMDS: Chinese, 853 speakers, 120 utterances per speaker, samplerate 16k
VCTK: English, 108 speakers, 123~502 uterances per speaker, samplerate 48k
Chinese transcriptions are converted to pinyin with tones by pypinyin. VCTK audios are downsampled to 16k with ffmpeg.
I'm training model on 8 v100 32G gpus. Parameters are:
	"balanced_sampling": true,
	"batch_size": 512,
	"case_sensitive": true,
	"characters": " !',-.?ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyzàáèéìíòóùúüāēěīńōūǎǐǒǔǘǚǜ",
	"checkpoint_each_epochs": 1,
	"dataset": "stcmds_vctk",
	"encoder_dimension": 256,
	"encoder_type": "generated",
	"epochs": 300,
	"generator_bottleneck_dim": 1,
	"generator_dim": 2,
	"languages": ["en", "zh"],
	"language_embedding_dimension": 0,
	"learning_rate": 0.005,
	"learning_rate_decay_each": 1000,
	"learning_rate_decay_start": 1000,
	"multi_language": true,
	"multi_speaker": true,
	"perfect_sampling": true,
	"predict_linear": false,
	"reversal_classifier": true,
    "reversal_classifier_dim": 256,
    "reversal_classifier_w": 0.125,
    "reversal_gradient_clipping": 0.25,
	"speaker_embedding_dimension": 1024,
	"version": "GENERATED-SWITCHING"
After training for 37 hours, logs are:
![eval](https://user-images.githubusercontent.com/24379054/125713423-5437ca6d-eda7-422a-a8ce-1e1de9f0f718.jpg)
![train](https://user-images.githubusercontent.com/24379054/125714594-09390b15-3f20-4fa9-aedd-89f2b0b974fd.jpg)
![image](https://user-images.githubusercontent.com/24379054/125714697-9c0e982c-b2e1-4162-ad9a-7f2505c4167a.png)
![image](https://user-images.githubusercontent.com/24379054/125714729-c5ceaf10-ebcb-44fa-9e50-a82785a11f93.png)

Eval/classifier seams to be too small, audios in "Audio/forced" and "Audio/generated" are very unclear。
Any advices please. Thanks in advance.

**Comment by mudong0419** [comment]
Some other parameters are:
    sample_rate = 16000                  # sample rate of source .wavs, used while computing spectrograms, MFCCs, etc.
    num_fft = 1024                       # number of frequency bins used during computation of spectrograms
    num_mels = 80                        # number of mel bins used during computation of mel spectrograms
    num_mfcc = 13                        # number of MFCCs, used just for MCD computation (during training)
    stft_window_ms = 50                  # size in ms of the Hann window of short-time Fourier transform, used during spectrogram computation
    stft_shift_ms = 12.5                 # shift of the window (or better said gap between windows) in ms
    griffin_lim_iters = 60               # used if vocoding using Griffin-Lim algorithm (synthesize.py), greater value does not make much sense
    griffin_lim_power = 1.5              # power applied to spectrograms before using GL
    normalize_spectrogram = True         # if True, spectrograms are normalized before passing into the model, a per-channel normalization is used
                                         # statistics (mean and variance) are computed from dataset at the start of training  
    use_preemphasis = True               # if True, a preemphasis is applied to raw waveform before using them (spectrogram computation)
    preemphasis = 0.97                   # amount of preemphasis, used if use_preemphasis is True
